recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "snowflake" %}
-{% set version = "0.11.0" %}
+{% set version = "0.12.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e85d9bc72fd4cac2d764bfb57032a1a9b51d514e34ddede74aeda7635f7fa5b4
+  sha256: 9a8c0fb1ac1a2720a0c913c3c1cd9151267f9d3c4373d2d2055a93281ec5ca28
   patches:
     # update 0.10.0: patch still needed
     - patches/0001-fix-pyproject.patch
```

Comment:
is this patch still needed for this version?

Reply:
patch seems to be still required. I updated the comment.

Reply:
thanks!